src/util/HeaderUtil.ts

```diff
@@ -495,3 +505,38 @@ export function parseForwarded(headers: IncomingHttpHeaders): Forwarded {
   }
   return forwarded;
 }
+
+/**
+ * Parses the link header(s) and returns an array of LinkEntry objects.
+ * @param linkHeaders - An array of linkHeaders (can be one header wrapped in an array)
+ * @returns A LinkEntry array, LinkEntry contains a link and a params Record&lt;string,string&gt;
+ */
+export function parseLinkHeader(linkHeaders: string[]): LinkEntry[] {
```

Comment:
```suggestion
export function parseLinkHeader(link: string | string[] = []): LinkEntry[] {
  const linkHeaders = Array.isArray(link) ? link : [ link ];
```
Seeing as this behaviour is also duplicated over the other classes.